### PR TITLE
style find-it link as a button; link location text to same url (excep…

### PR DESCRIPTION
**Pull request by peetucket**
…t when not available or when an external library)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -59,6 +59,7 @@ group :development, :test do
   # We use database cleaner to empty out the database between tests (see spec_helper for usage)
   gem 'database_cleaner'
   gem 'poltergeist', '>= 1.8.1'
+  gem 'launchy' # useful for debugging rspec/capybara integration tests -- put "save_and_open_page" in your test to debug
```
Comment (jkeck) [inline Gemfile]: :+1: 
Can we separate the addition of this dependency into a separate commit?  That way if one or the other needs to be reverted it won't inadvertently remove something else.

File: app/assets/stylesheets/searchworks-mixins.css.scss [hunk 2]
```diff
@@ -95,13 +95,18 @@
 }
 
 @mixin stackmap-find-it {
-  margin-left: 10px;
-
+  background-color: #007cbc;
+  border-color: #006ba3;
+  margin-right: 5px;
+  color: #ffffff;
```
Comment (jkeck) [inline app/assets/stylesheets/searchworks-mixins.css.scss]: Can we move these colors into variables (or use existing ones when available)?  I believe that is the preferred practice in the scss world and will help us a lot when we implement SULStyles.

(FWIW `#ffffff` is already defined as `$white`)

**Comment by jkeck** [comment]
:+1: LGTM! Few super-minor comments.

File: Gemfile (hunk 1 above)
Comment (peetucket) [inline Gemfile]: :+1: sounds good

**Pull request merged**